platform/platform-impl/src/com/intellij/openapi/editor/impl/EditorComponentImpl.java

```diff
@@ -362,7 +364,7 @@ public void putInfo(@NotNull Map<? super String, ? super String> info) {
   @SuppressWarnings("UnusedReturnValue")
   @RequiresEdt
   private boolean setupEditorSwingCaretUpdatesCourierIfRequired() {
-    if ((myEditorSwingCaretUpdatesCourier != null) || (myEditor == null) || (myEditor.isDisposed()) ) {
+    if ((myEditorSwingCaretUpdatesCourier != null) || (myEditor == null) || (myEditor.isDisposed())) {
```

Comment:
I would kindly ask you to revert the formatting and non-code changes in the PR. I understand that this is probably the result of running auto-formatting in the IDE, but we try to keep commits isolated, so that they contain only changes that are necessary for the fix. You can look into the following page for more details on commit guidelines in our repo: https://plugins.jetbrains.com/docs/intellij/intellij-coding-guidelines.html#commits.